docs/store-submit/huawei-app-gallery.md

```diff
@@ -49,7 +49,7 @@ Then enter a user-friendly name to identify the key in the lists and press save.
 
 Your upload keystore file should also be uploaded to Huawei AppGallery. Follow the [AppGallery documentation](https://developer.huawei.com/consumer/en/doc/development/AppGallery-connect-Guides/agc-appsigning-newapp-0000001052418290#EN-US_TOPIC_0000001052418290__section1959661616436) to convert your keystore file and upload it to `App Signing` section on AppGallery.
 
-<Screenshot url='https://cdn.appcircle.io/docs/assets/huawei-8signing.png' />
+<ExternalScreenshot url='https://cdn.appcircle.io/docs/assets/huawei-8signing.png' width='3482px' height='1558px' />
```

Comment:
I tried to understand the case in this sample. I modified the "max-heigth" in inspector and it fits.

![test-max-height-360](https://github.com/appcircleio/appcircle-docusaurus/assets/7355362/b2e74863-d0fc-41ee-ab6b-02029b0e7147)

IMO, maximum height should also be controlled by the contributor who is adding the image to the documents.

Below is the same page on production.

![Screenshot from 2023-11-28 07-14-01](https://github.com/appcircleio/appcircle-docusaurus/assets/7355362/5b5b432f-df48-498b-bb09-8c64e9a3c287)